docs/plugins/trello-github-integ_plugin.md

```diff
@@ -37,13 +24,51 @@ tools:
     owner: YOUR_GITHUB_USERNAME
     # the repo where you'd like to setup GitHub Actions; please change the value below.
     repo: YOUR_REPO_NAME
-    # integration tool name
-    api:
-      name: trello
-      # name of the Trello kanban board
-      kanbanBoardName: kanban-name
+    # reference from dependency
+    boardId: ${{ default.trello.outputs.boardId }}
```

Comment:
At this point, the user hasn't read the "outputs" part yet, so they might be confused by this syntax. Maybe we can:
- either add a comment to point it out that this is referring to outputs, which is explained later;
- or use some placeholders